src/Rbac.php

```diff
@@ -22,20 +25,19 @@ class Rbac extends AbstractIterator
     protected $createMissingRoles = false;
 
     /**
-     * @param  bool                     $createMissingRoles
+     * @param  bool $createMissingRoles
      * @return \Zend\Permissions\Rbac\Rbac
      */
-    public function setCreateMissingRoles($createMissingRoles)
+    public function setCreateMissingRoles(bool $createMissingRoles): Rbac
```

Comment:
We are using single space before colon in return type declaration.

Reply:
Also as a return type I would use `self` (it's clearer when you read class in the middle then we know straight away that we return the same class).

Reply:
Or we should drop the fluent interface altogether, as we have in other components, and return `void`. (This comment applies to all setter methods that currently implement a fluent interface.)

Reply:
I removed fluent interface in all the library.